lib/Lifters/FunctionLifter.cpp

```diff
@@ -1175,12 +1183,18 @@ void FunctionLifter::VisitInstructions(uint64_t address) {
         maybe_decl = std::move(inst_func.value());
       }
 
+      auto is_noreturn = maybe_decl->is_noreturn;
       llvm::IRBuilder<> ir(block);
       auto new_mem_ptr = CallCallableDecl(
           block, options.program_counter_init_procedure(ir, pc_reg, redir_addr),
           std::move(maybe_decl.value()));
 
-      auto ret = ir.CreateRet(new_mem_ptr);
+      llvm::Instruction* ret;
+      if(is_noreturn) {
+        ret = ir.CreateUnreachable();
```

Comment:
Terminating tail call to `__remill_error`.

Reply:
What should I return after the tail call?